.circleci/config.yml

```diff
@@ -727,20 +727,23 @@ commands:
     steps:
       - run: python .circleci/scripts/wait_for_master.py <<parameters.scheme>>://<<parameters.host>>:<<parameters.port>>
 
-  upload-cloud-watch-to-s3:
+  locate-cloud-watch-logs:
     parameters:
       cluster-id:
         type: string
-      bucket:
+      region:
         type: string
-        default: det-ci-cloud-watch-logs
+        default: us-west-2
     steps:
       - run:
-          name: Upload cloud watch logs S3
+          name: Locate cloud watch logs
           when: always
           command: |
             LOG_GROUP=$(python .circleci/scripts/get_output_from_stack.py <<parameters.cluster-id>> LogGroup)
-            python .circleci/scripts/upload_cloud_watch_to_s3.py ${LOG_GROUP} <<parameters.bucket>> <<parameters.cluster-id>>
+            echo "Cluster logs can be found in Cloudwatch under the log group $LOG_GROUP, or the URL below:"
+            ENC_LOG_GROUP=$(echo $LOG_GROUP | sed "s/\//\$252F/g")
+            echo "https://<<parameters.region>>.console.aws.amazon.com/cloudwatch/home?region=<<parameters.region>>#logsV2:log-groups/log-group/$ENC_LOG_GROUP"
```

Comment:
This is an imprecise science. These URLs are not an API and may break at any time, but it's just for convenience.